app/_locales/en/messages.json

```diff
@@ -2453,10 +2461,10 @@
     "message": "WARNING: Never disclose your Secret Recovery Phrase. Anyone with this phrase can take your Ether forever."
   },
   "secretPhrase": {
-    "message": "Only the first account on this wallet will auto load. After completing this process, to add additional accounts, click the drop down menu, then select Create Account."
+    "message": "If you restore a wallet using the Secret Recovery Phrase, only the wallet’s initial account might appear at first. You'll be able to $1 created previously."
```

Comment:
hmm interesting, @mcmire. The "might" was actually referring to the case when **only** the first account shows up even though the user had created more accounts in the past. 
The decision to use "might" here is because we do have a method to detect if other accounts have value and show those too, but it doesn't cover all cases (e.g. the user has other accounts but they only have value on other networks; or the 2nd and 3rd account don't have value but the 4th does, but none of those will show up).

Any suggestion on how we could improve this to make it sound less risky?

We were saying "...only the wallet’s initial account **will** appear at first." previously but that was technically incorrect as Mark pointed out [here](https://consensys.slack.com/archives/GTQAGKY5V/p1643402191820629?thread_ts=1643401198.490789&cid=GTQAGKY5V).

Reply:
Ah I forgot about Mark's comment. I am not 100% sure that this copy addresses everything he is saying. May I update the Notion doc or is it too late?

Reply:
Yes, feel free to update it @mcmire! 
One thing to keep in mind is that this is just quick fix while we work on the [new 'Forgot password' flow](https://www.figma.com/file/8z4Csc3wlOIkUz2OP1eTZW/Secure-UX-Improvements?node-id=249%3A6059), but the more we can get right with this copy update, the better!

Reply:
I updated the page in Notion. I decided not to update the copy directly since I know multiple people worked on this. I'll ping in Slack but you should be able to see what I wrote there.

Reply:
Hey @NiranjanaBinoy sorry for the delay on this. We were reviewing this copy with Oliver. Here is the final copy:

Reset Wallet

MetaMask does not keep a copy of your password. If you’re having trouble unlocking your account, you will need to reset your wallet. You can do this by providing the Secret Recovery Phrase you used when you set up your wallet.

This action will delete your current wallet and Secret Recovery Phrase from this device, along with the list of accounts you’ve curated. After resetting with a Secret Recovery Phrase, you’ll see a list of accounts based on the Secret Recovery Phrase you use to reset. This new list will automatically include accounts that have a balance. You’ll also be able to [re-add any other accounts](https://metamask.zendesk.com/hc/en-us/articles/360015489271-How-to-add-missing-accounts-after-restoring-with-Secret-Recovery-Phrase) created previously. Custom accounts that you’ve imported will need to be [re-added](https://metamask.zendesk.com/hc/en-us/articles/360015489331-How-to-import-an-Account), and any custom tokens you’ve added to an account will need to be [re-added](https://metamask.zendesk.com/hc/en-us/articles/360015489031-How-to-add-unlisted-tokens-custom-tokens-in-MetaMask) as well.

Make sure you’re using the correct Secret Recovery Phrase before proceeding. You will not be able to undo this.

Reply:
Thanks, @bschorchit; I will update the changes.